pkg/providers/tinkerbell/upgrade.go

```diff
@@ -371,6 +371,39 @@ func (p *Provider) isScaleUpDown(oldCluster *v1alpha1.Cluster, newCluster *v1alp
 	return false
 }
 
+func (p *Provider) isEksaVersionUpgrade(currentSpec, newClusterSpec *cluster.Spec) bool {
+	return currentSpec.Bundles.Spec.Number != newClusterSpec.Bundles.Spec.Number
```

Comment:
I think this can be a variable at the site of use given its only used in 1 place. 

E.g. 

```
isEKSAVersionUpgrade := currentSpec.Bundles.Spec.Number != newClusterSpec.Bundles.Spec.Number
```